Make-a-pull-request.md

```diff
@@ -193,4 +236,26 @@ git push origin --delete new-branch-name
 
 ## Step 8: Celebrate
 
-Congratulations, you have contributed to Oppia! To learn some more advanced ways to use PRs at Oppia, see our [[guide to common PR workflows|Common-pull-request-workflows]].
+Congratulations, you have successfully contributed to Oppia! 🎉
+
+### Tips for Managing Multiple PRs
+
+1. **Prioritize Existing PRs**:
+   - Always prioritize following up on your existing PRs and getting them ready for the next round of review. Only start working on new issues/PRs after **all** the other PRs you're working on are in the "waiting for review" stage. This helps avoid creating a logjam in the build queues.
+
+2. **Create Separate Branches**:
+   - Always create a new branch for each pull request. This helps keep your changes isolated and makes it easier to manage multiple PRs.
+
+3. **Rebase Frequently**:
+   - Regularly rebase your branches with the latest changes from the `develop` branch to avoid merge conflicts.
```

Comment:
Actually, we don't generally rebase at Oppia. We must have forgotten to update this. Instead, let's say:

```suggestion
3. **Update Branches Frequently**:
   - Regularly pull in changes from the `develop` branch to your feature branch. This will help you avoid merge conflicts.
```

Reply:
You're right. I've already updated it.